Doc/library/configparser.rst

```diff
@@ -1,1368 +1,1371 @@
-      set to true) for *internal* storage of non-string values, full
-      functionality (including interpolation and output to files) can only be
-      achieved using string values.
-
-      This method lets users assign non-string values to keys internally.  This
-      behaviour is unsupported and will cause errors when attempting to write
-      to a file or get it in non-raw mode.  **Use the mapping protocol API**
-      which does not allow such assignments to take place.
-
-
-Exceptions
-----------
-
-.. exception:: Error
-
-   Base class for all other :mod:`configparser` exceptions.
-
-
-.. exception:: NoSectionError
-
-   Exception raised when a specified section is not found.
-
-
-.. exception:: DuplicateSectionError
-
-   Exception raised if :meth:`add_section` is called with the name of a section
-   that is already present or in strict parsers when a section if found more
-   than once in a single input file, string or dictionary.
-
-   .. versionadded:: 3.2
-      Optional ``source`` and ``lineno`` attributes and arguments to
-      :meth:`__init__` were added.
-
-
-.. exception:: DuplicateOptionError
-
-   Exception raised by strict parsers if a single option appears twice during
-   reading from a single file, string or dictionary. This catches misspellings
-   and case sensitivity-related errors, e.g. a dictionary may have two keys
-   representing the same case-insensitive configuration key.
-
-
-.. exception:: NoOptionError
-
-   Exception raised when a specified option is not found in the specified
-   section.
-
-
-.. exception:: InterpolationError
-
-   Base class for exceptions raised when problems occur performing string
-   interpolation.
-
-
-.. exception:: InterpolationDepthError
-
-   Exception raised when string interpolation cannot be completed because the
-   number of iterations exceeds :const:`MAX_INTERPOLATION_DEPTH`.  Subclass of
-   :exc:`InterpolationError`.
-
-
-.. exception:: InterpolationMissingOptionError
-
-   Exception raised when an option referenced from a value does not exist.
-   Subclass of :exc:`InterpolationError`.
-
-
-.. exception:: InterpolationSyntaxError
-
-   Exception raised when the source text into which substitutions are made does
-   not conform to the required syntax.  Subclass of :exc:`InterpolationError`.
-
-
-.. exception:: MissingSectionHeaderError
-
-   Exception raised when attempting to parse a file which has no section
-   headers.
-
-
-.. exception:: ParsingError
-
-   Exception raised when errors occur attempting to parse a file.
-
-   .. versionchanged:: 3.2
-      The ``filename`` attribute and :meth:`__init__` argument were renamed to
-      ``source`` for consistency.
-
-
-.. rubric:: Footnotes
-
-.. [1] Config parsers allow for heavy customization.  If you are interested in
-       changing the behaviour outlined by the footnote reference, consult the
-       `Customizing Parser Behaviour`_ section.
+:mod:`configparser` --- Configuration file parser

```

Comment:
The diff looks as if you changed everything in the file - is there a whitespace issue?

It should show only what you changed.

Reply:
Yeah. There is some whitespace issue. So I again tried upstreaming from the main branch and redo the modification. But still it shows the same.

Reply:
You need to sort this out in the PR. It won't be merged as it is now.

Reply:
Closing this pull request and creating new one. Sorry for the inconvenience.

New one: https://github.com/python/cpython/pull/31413